.github/workflows/scheduled.yml

```diff
@@ -43,22 +41,34 @@ jobs:
 
       - name: Generate matrix
         id: matrix
+        working-directory: ${{ steps.set-up-homebrew.outputs.repository-path }}
         run: |
-          formula="$(find "$(brew --repo "${TAP}")/Formula" -type f | shuf -n "${TEST_COUNT}" | xargs -I{} basename {} .rb)"
-          # shellcheck disable=SC2086
+          # Index all formulae so that we test:
+          #   - formulae 1 to 50 on the first day of the year
+          #   - formulae 51 to 100 on the second day of the year
+          #   - formulae 101 to 150 on the third day of the year
+          #   - ...
+          # This works fine as long as we have fewer than 365 * TEST_COUNT formulae.
```

Comment:
Would be good to make this error if that happens.

Reply:
Done!